python/pyspark/streaming/kafka.py

```diff
@@ -104,7 +104,7 @@ def createDirectStream(ssc, topics, kafkaParams, fromOffsets=None,
         :param topics:  list of topic_name to consume.
         :param kafkaParams: Additional params for Kafka.
         :param fromOffsets: Per-topic/partition Kafka offsets defining the (inclusive) starting
-                            point of the stream.
+                            point of the stream (Dict with keys of type TopicAndPartition and int values).
```

Comment:
I would say sth like ``a dictionary containing `TopicAndPartition` to integers.``.

Reply:
Yeah, that works too.

Reply:
Shall we fix as so?